### PR TITLE
add `listRepositories`, `createRepository`, `deleteRepository` to `CodeCommit` client

### PR DESCRIPTION
**Pull request by nealio82**
This'll stop me having to mix using AsyncAWS and the official AWS SDK for these particular operations 😛

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -92,10 +92,13 @@
             "example": "https://raw.githubusercontent.com/aws/aws-sdk-php/${LATEST}/src/data/codecommit/2015-04-13/examples-1.json",
             "api-reference": "https://docs.aws.amazon.com/codecommit/latest/APIReference",
             "methods": [
+                "CreateRepository",
```
Comment (jderusse) [inline manifest.json:95]: could you add an entry in the Changelog of `CodeCommit` for these new operations?

Comment (nealio82) [reply manifest.json:95]: @jderusse done 👍

File: src/Service/CodeCommit/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,14 @@
 
 ## NOT RELEASED
 
+## 0.2.0
+
```
Comment (jderusse) [inline src/Service/CodeCommit/CHANGELOG.md:6]:
```suggestion
```

**Comment by jderusse** [comment]
thank you @nealio82

File: src/Service/CodeCommit/CHANGELOG.md (hunk 2 above)
Comment (nealio82) [reply src/Service/CodeCommit/CHANGELOG.md]: Ohhhh, I see. Ok :)

**Pull request merged**